src/r3d.imageio/r3dinput.cpp

```diff
@@ -148,7 +148,8 @@ OIIO_EXPORT int r3d_imageio_version = OIIO_PLUGIN_VERSION;
 OIIO_EXPORT const char*
 r3d_imageio_library_version()
 {
-    return "R3D";
+    // Note: SDK version can differ from the actual library loaded
+    return "R3D 8.5.1";
```

Comment:
Can you get this from the library or the r3dsdk headers? This will obviously be wrong as soon as a new release is out.

Reply:
I can get this from the CMake script. Indeed, hardcoding the version string is not proper way to handle this. The actual library version may be different at runtime, as I've tested so far. But nothing exploded. This version is reported just after initialization from R3DSDK::GetSdkVersion().